### DISCUSSION TITLE
Welcome to PlantUmlClassDiagramGenerator Discussions!

### DISCUSSION THREAD

**Discussion by pierre3** [pierre3/PlantUmlClassDiagramGenerator#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by markcollinscope** [comment]
Perhaps an odd question. I've installed C# to plantuml in Visual Studio Code. Great. But how do I get it to generate something? Nothing seems to be happening.  Assistance appreciated. Perhaps it's not set up correctly?

**Comment by pierre3** [comment]
Thank you for reaching out! Please check the logs using the method outlined in the link below, and if you see any errors or issues, please let us know the details.

https://github.com/pierre3/PlantUmlClassDiagramGenerator/discussions/70

**Comment by Hojou** [comment]
Hi and thanks for this great tool. It is so nice to be able to visualize your solution automatically!
I have a day job as a web developer, but in my spare time i like to dabble in game development, using Unity.

My idea:
Would it be possible to make a switch to exclude interface references, like my classes implement INetworkSerializable or derrving from a generic (Unity) baseclass like MonoBehaviour. When you have a lot of classes all deriving from something or implemeting something, it clutters up the view. Arrows/lines flying all over the place.

As you can see, even when using `[PlantUmlIgnoreAssociation]` (to make some data relations go away) i can't make MonoBehavior, NetworkBehaviour, ScriptableObject and INetworkSerializable go away
![image](https://github.com/pierre3/PlantUmlClassDiagramGenerator/assets/514426/56f6c3ad-fb11-4b8b-9a6a-729141a3c901)

**Comment by Hojou** [comment]
Also in the lines of my previous post. Would it be possible to somehow omit any reference to any external class. In my Project i use a type `Vector2Int` and `Vector2` a lot. It is just a simple class Unity provides for me that is just two integer values (x and y). Every time a class references that, i also get arrows pointing to that in the diagram. That is a lot of arrows. I know i can manually remove these references, which is what i do now - so the above problem is bigger, but this would be nice to not have to manually remove any references to "simple data types"

So perhaps if we could omit whole classes, interfaces, namespaces (that would probably solve both this issue and the above one)
